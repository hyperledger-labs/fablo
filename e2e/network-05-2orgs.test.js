const { performTests } = require('./performTests');

const label = 'network-04-2orgs';

describe(label, () => {
  performTests(label, 'fabrikkaConfig-2orgs-2channels-1chaincode.json');
});
