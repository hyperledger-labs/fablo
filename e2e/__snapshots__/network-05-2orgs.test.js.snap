// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`samples/fablo-config-hlf1.3-2orgs-1chaincode-private-data.json should create proper files from samples/fablo-config-hlf1.3-2orgs-1chaincode-private-data.json 1`] = `Array []`;
